plugins/wrangler/api_token.go

```diff
@@ -0,0 +1,52 @@
+package wrangler
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://developers.cloudflare.com/workers/wrangler/system-environment-variables/"),
+		ManagementURL: sdk.URL("https://dash.cloudflare.com/profile/api-tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.AccountID,
+				MarkdownDescription: "The account ID for the Workers related account, can be found in the Cloudflare dashboard, can usually be inferred by Wrangler.",
+				Secret:              true,
```

Comment:
I don't think this is secret, right? Making it secret will also make it concealed in the 1Password GUI.
```suggestion
				Secret:              false,
```